CHANGELOG.md

```diff
@@ -42,6 +42,10 @@ Bottom level categories:
 
 ### Changes
 
+#### General
```

Comment:
We should also make a major changes note about the api of `describe()` changing.